tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -57,8 +89,10 @@ public ExporterSettings(IConfigurationSource source)
 
             if ((partialFlushMinSpans ?? 0) <= 0)
             {
-                PartialFlushMinSpans = 500;
+                partialFlushMinSpans = 500;
             }
+
+            PartialFlushMinSpans = partialFlushMinSpans.Value;
```

Comment:
This was never set since it was introduced. 
It's an unrelated bug-fix I caught with added tests. 
Happy to separate this out but it seemed innocuous enough.